services/storage/src/simcore_service_storage/dsm.py

```diff
@@ -115,30 +131,36 @@ class DataStorageManager:
         ping_ok = await self.ping_datcore(user_id=user_id)
         if ping_ok:
             datcore = {
-            "name" : DATCORE_STR,
-            "id"   : DATCORE_ID
+                "name" : DATCORE_STR,
+                "id"   : DATCORE_ID
             }
             locs.append(datcore)
 
         return locs
 
-    # pylint: disable=R0201
-    def location_from_id(self, location_id : str):
+    @classmethod
+    def location_from_id(cls, location_id : str):
         return _location_from_id(location_id)
 
-    async def ping_datcore(self, user_id: str):
-        api_token, api_secret = await self._get_datcore_tokens(user_id)
+    async def ping_datcore(self, user_id: str) -> bool:
+        """ Checks whether user account in datcore is accesible
+
+        :param user_id: user identifier
+        :type user_id: str
+        :return: True if user can access his datcore account
+        :rtype: bool
+        """
```

Comment:
Is this the official osparc way for commenting? Is that auto generated?

Reply:
I added a plugin to auto generate as much as possible